src/pages/patientView/mutationalSignatures/MutationalSignaturesContainer.tsx

```diff
@@ -34,8 +34,14 @@ export default class MutationalSignaturesContainer extends React.Component<
         // split the id by "_", the last part is the version info
         // we know split will always have results
         // use uniq function to get all unique versions
-        return _.chain(this.props.profiles)
-            .map(profile => _.last(profile.molecularProfileId.split('_'))!)
+        // not all patients have the newest mutational signatures --> check if data is present before giving the options
+        let possibleOptions = this.props.profiles.map(
+            profile => _.last(profile.molecularProfileId.split('_'))!
+        );
+        possibleOptions = possibleOptions.filter(item =>
+            Object.keys(this.props.data).includes(item)
+        );
+        return _.chain(possibleOptions)
```

Comment:
this can just be _.uniq(possibleOptions) and then you don't need _.chain or _.value.